### PR TITLE
README: add some desc & how to build/develop

### PR DESCRIPTION
**Pull request by 6543**
as title

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#/bin/bash
+
+yarn ng serve &
+pid="$!"
+sleep 5s
```
Comment (crapStone) [inline run.sh:5]: It could run much longer than 5 seconds

**Review by crapStone — COMMENTED** [review]
Do we really need this script?

I think a simple explanation in the README is enough.

**Comment by Epsilon02** [comment]
I am with @crapStone on this one. In my opinion is useless for me for development because I need both outputs in different terminals for better overview whats going on.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,18 @@
...
+Frontend is written in angular (JS) and backend is in pure rust.
+
+To start execute `yarn ng serve` to build ans serve UI,
+execute `yarn tauri-start` to get the aplication window.
+
+## Build
+
+you need the tauri-builder. Get it  via `cargo install tauri-bundler`.
+
+Just exec `yarn tauri build` afterwards.
```
Comment (6543) [inline README.md:18]: looks like `yarn build` is needed here?

Comment (6543) [inline README.md:18]:
```suggestion
Just exec `yarn build && yarn tauri build` afterwards.
```

File: README.md [hunk 3]
```diff
@@ -1,4 +1,18 @@
...
-> Coming soon...
+Generate "experiment wise safety sheets" in compliance to European law.
 
+## Develop
+
+CaBr2 is a [tauri](https://github.com/tauri-apps/tauri) project.
+
+Frontend is written in angular (JS) and backend is in pure rust.
+
+To start execute `yarn ng serve` to build ans serve UI,
```
Comment (6543) [inline README.md:11]:
```suggestion
To start execute `yarn start` to build ans serve UI,
```

File: README.md [hunk 4]
```diff
@@ -1,4 +1,18 @@
 # CaBr2
 
-> Coming soon...
+Generate "experiment wise safety sheets" in compliance to European law.
 
+## Develop
+
+CaBr2 is a [tauri](https://github.com/tauri-apps/tauri) project.
+
+Frontend is written in angular (JS) and backend is in pure rust.
```
Comment (Epsilon02) [inline README.md:9]:
```suggestion
Frontend is written in Angular 11 and backend is in pure rust.
```
AngularJS is another Framework.

File: README.md [hunk 5]
```diff
@@ -1,4 +1,18 @@
 # CaBr2
 
-> Coming soon...
+Generate "experiment wise safety sheets" in compliance to European law.
 
+## Develop
+
+CaBr2 is a [tauri](https://github.com/tauri-apps/tauri) project.
```
Comment (crapStone) [inline README.md:7]:
```suggestion
CaBr2 is a [tauri](https://tauri.studio) project.
```

File: README.md [hunk 6]
```diff
@@ -1,4 +1,18 @@
...
+
+Frontend is written in Angular 11 and backend is in pure rust.
```
Comment (crapStone) [inline README.md:9]:
```suggestion
Frontend is written in Angular 11 and backend is in rust.
```

File: README.md [hunk 7]
```diff
@@ -1,4 +1,18 @@
...
+
+To start execute `yarn start` to build ans serve UI,
+execute `yarn tauri-start` to get the aplication window.
+
+## Build
+
+you need the tauri-builder. Get it  via `cargo install tauri-bundler`.
```
Comment (crapStone) [inline README.md:16]: `-f` would also update the installed binary if it is outdated
```suggestion
you need the tauri-builder. Get it  via `cargo install -f tauri-bundler`.
```

**Review by crapStone — APPROVED** [review]

**Pull request merged**